playbooks/aws/openshift-cluster/provision.yml

```diff
@@ -0,0 +1,264 @@
+---
+- name: Setup the vpc and the master node group
+  hosts: localhost
+  connection: local
+  gather_facts: yes
+  tasks:
+  - name: get provisioning vars
+    include_vars: vars.yml
+
+  - name: create default vpc
+    include_role:
+      name: openshift_aws_vpc
+    vars:
+      r_openshift_aws_vpc_clusterid: "{{ provision.clusterid }}"
+      r_openshift_aws_vpc_cidr: "{{ provision.vpc.cidr }}"
+      r_openshift_aws_vpc_subnets: "{{ provision.vpc.subnets }}"
+      r_openshift_aws_vpc_region: "{{ provision.region }}"
+      r_openshift_aws_vpc_tags: "{{ provision.vpc.tags }}"
+
+  - name: create aws ssh keypair
+    include_role:
+      name: openshift_aws_ssh_keys
+    vars:
+      r_openshift_aws_ssh_keys_users: "{{ provision.instance_users }}"
+      r_openshift_aws_ssh_keys_region: "{{ provision.region }}"
+
+  - name: create s3 bucket for registry
```

Comment:
don't think the boundary for a role is based on size.  I base roles on the work that it accomplishes.